### ISSUE TITLE
non english language does not search matching title

### ISSUE THREAD

**Issue by perflow** [elgatito/plugin.video.elementum#77]
selecting a title in elementum does not provide matching title results

I've looked at the log file and it search in non english language which gives different title results,
I think quasar resolve link by title number

elemuntum log: https://pastebin.com/rFnXYhkW

quasar log: https://pastebin.com/yHdscsXC

**Comment by elgatito** [comment]
@NeoCro44 including in the issue.

**Comment by elgatito** [comment]
This is not a bug with Elementum, but a bug/misbehavior coming from Quasar.

There is an option to use original titles, that is used to identify whether we pass original movie/show title to Burst (or other searchers). But it was not working, original title was used anyway (https://github.com/scakemyer/quasar/blob/master/providers/xbmc.go#L123).

I have fixed it for Elementum, now it's sending localized title to searcher if Use Original Title is not selected.

But that caused a new case, some trackers contain only original/english name in the title, some use only french and so on. So it your Kodi language is English - you are fine, if not - your main search query is localized.

So I've pushed new Burst version, where some trackers' definitions are changed to use original title always. 

PS: I can't change all the trackers as most of them are private, but if you find another trackers which need to be changed - feel free to create an issue in Burst repository.
